react-carousel/my-app/src/ArrowButton.tsx

```diff
@@ -0,0 +1,13 @@
+import { FaChevronLeft, FaChevronRight } from 'react-icons/fa';
+
+type Props = {
+  type: string;
```

Comment:
It would be nice to type this as `type: 'back' | 'next'`

That way TypeScript can make sure you don't misspell the words.